### DISCUSSION TITLE
Want to leave a note that all sv3c cameras are supported

### DISCUSSION THREAD

**Discussion by Darkyputz** [blakeblackshear/frigate#15394] [Ideas]
Hello.

I see alot of cams in your supported cams list with features and i would like to share that all my sv3c cams are supported.
What would be the best format to share what cams and what function?
They all do rtsp and onvif and my pan and tilt cam also works with frigates pan and tilt function besides the autotracking, but maybe i need to learn some more about that feature

**Comment by peteDDD** [comment]
Thank you.  Have you had success configuring with go2rtc as well?  Could you share your config file?

**Reply by Darkyputz** [reply]
Hello, yes, I have them configured with ptz, go2rtc and everythig that I wanted besides follow function but they can't do it since they are only onvif profile s
Go2rtc config looks like this:
`go2rtc:
  streams:
    porch_cam:
      - rtsp://192.168.178.41/11
      - ffmpeg:porch_cam#audio=aac
    porch_cam_sub:
      - rtsp://192.168.178.41/12
  `
Let me know it you need more info
